packages/dendron-next-server/pages/workspace/config.tsx

```diff
@@ -0,0 +1,326 @@
+                              onChange={(e) =>
+                                setFieldValue(name, e.target.checked)
+                              }
+                              colorScheme="positive"
+                            />
+                          )}
+                        </Field>
+                      </Stack>
+
+                      <FormHelperText>
+                        Pretty refs help you identify when content is embedded
+                        from elsewhere and provide links back to the source.
+                      </FormHelperText>
+                    </FormControl>
+
+                    <FormControl
+                      isInvalid={
+                        !!(touched.site?.copyAssets && errors.site?.copyAssets)
+                      }
+                    >
+                      <Stack direction="row" align="center">
+                        <FormLabel margin={0}>Copy assets?</FormLabel>
+
+                        <Field name="site.copyAssets">
+                          {({
+                            form: { setFieldValue },
+                            field: { name, value },
+                          }) => (
+                            <Switch
+                              id={name}
+                              name={name}
+                              value={value}
+                              isChecked={values?.site?.copyAssets}
+                              onChange={(e) =>
+                                setFieldValue(name, e.target.checked)
+                              }
+                              colorScheme="positive"
+                            />
+                          )}
+                        </Field>
+                      </Stack>
+
+                      <FormHelperText>
+                        If enabled, assets will be copied from the vault to the
+                        site.
+                      </FormHelperText>
+                    </FormControl>
+                  </Stack>
+                </Box>
+
+                <Box as="fieldset">
+                  <Heading size="md" as="legend">
+                    Site hierarchies
+                  </Heading>
+
+                  <Stack spacing={4}>
+                    {configData.site?.siteHierarchies?.map(
+                      (hierarchy, index) => (
+                        <Tooltip
+                          label="Site hierarchies are not yet editable via this form."
+                          placement="top"
+                          hasArrow
+                        >
+                          <FormControl as={Stack} direction="row" spacing={2}>
+                            <Text
+                              htmlFor={`site.siteHierarchies[${index}]`}
+                              whiteSpace="nowrap"
+                              paddingTop={2} // align with text field
+                            >
+                              Hierarchy {index}
```

Comment:
same comment as vaults. no need for an index at this point

Reply:
:heavy_check_mark: @ 2177b0f2